subsys/bluetooth/shell/Makefile

```diff
@@ -1,2 +1,3 @@
 obj-y += bt.o
 obj-y += gatt.o
+obj-y += ticker.o
```

Comment:
This seems to be missing some conditionality. "obj-$(CONFIG_BLUETOOTH_CONTROLLER) += ticker.o" perhaps?